certora/Auxiliar.sol

```diff
@@ -0,0 +1,81 @@
+pragma solidity 0.8.9;
+
+import "../src/WormholeGUID.sol";
+
+interface OracleLike {
+    function signers(address) external view returns (uint256);
+}
+
+contract Auxiliar {
```

Comment:
Maybe add some brief comment stating the purpose of this contract?

Reply:
Done!